olive/platform_sdk/qualcomm/snpe/configure.py

```diff
@@ -9,20 +9,20 @@
 from pathlib import Path
 
 from olive.common.utils import run_subprocess
-from olive.snpe.utils.local import get_snpe_root, get_snpe_target_arch, get_snpe_win_arch_name
+from olive.platform_sdk.qualcomm.constants import SDKTargetDevice
+from olive.platform_sdk.qualcomm.snpe.env import SNPESDKEnv
 
-logger = logging.getLogger("olive.snpe.configure")
+logger = logging.getLogger(__name__)
 
 
 def dev(args):
-    snpe_arch = get_snpe_target_arch(False)
-    if snpe_arch != "x64-Linux":
+    snpe_env = SNPESDKEnv().env
+    snpe_arch = snpe_env.get("TARGET_ARCH")
```

Comment:
it seems multiple place need get the target_arch, better to give a helper function by just wrapper the TARGET_ARCH querying logic.

Reply:
done.

Reply:
remove TARGET_ARCH from env. Get it by instance attribute.